main.go

```diff
@@ -86,8 +83,16 @@ func main() {
 	}
 	defer conversion.Close(lf)
 
+	if schemaOnly && dataOnly {
```

Comment:
Aside (and not suggesting any change in the PR): I find the dataOnly and schemaOnly booleans very confusing in the logic of the rest of this function! Double negatives like "if !schemaOnly" and "if !dataOnly" are hard to keep straight.

I think flag names are correct (and correspond to similar flags in mysqldump and pg_dump).
However, internally, I think we should translate them to internal booleans like "processSchema" and "processData". That would make the code a bit more self-evident.

Reply:
I also had a hard time reading through the code due to double negatives. I think once we nail down on the submodule structure of commands and refactor towards the same, then many of these issues should go away since I expect separate modules for processSchema and processData then.